File: app/src/main/java/com/example/tipper/Graph.java
package com.example.tipper;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.jjoe64.graphview.GraphView;
import com.jjoe64.graphview.GridLabelRenderer;
import com.jjoe64.graphview.LegendRenderer;
import com.jjoe64.graphview.helper.StaticLabelsFormatter;
import com.jjoe64.graphview.series.DataPoint;
import com.jjoe64.graphview.series.PointsGraphSeries;

public class Graph extends AppCompatActivity {
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.graph_layout);
        GraphView graphView = (GraphView) findViewById(R.id.graphView);
        StaticLabelsFormatter staticLabelsFormatter = new StaticLabelsFormatter(graphView);
        PointsGraphSeries<DataPoint> series = new PointsGraphSeries<DataPoint>(new DataPoint[]{
                new DataPoint(0, 25),
                new DataPoint(1, 22),
                new DataPoint(2, 24),
                new DataPoint(3, 26),
                new DataPoint(4, 21),
                new DataPoint(5, 20),
                new DataPoint(6, 23),
        });
        graphView.addSeries(series);
        staticLabelsFormatter.setHorizontalLabels(new String[]{"Jan", "Feb", "Mar", "Apr", "May", "Jun"});

        GridLabelRenderer gridLabelRenderer = graphView.getGridLabelRenderer();
        gridLabelRenderer.setLabelFormatter(staticLabelsFormatter);
        series.setColor(R.color.colorPrimaryDark);
        graphView.setTitle(getString(R.string.graph_title));
        graphView.getLegendRenderer().setVisible(true);
        graphView.getLegendRenderer().setAlign(LegendRenderer.LegendAlign.BOTTOM);
        gridLabelRenderer.setHorizontalAxisTitle(getString(R.string.month));
        gridLabelRenderer.setVerticalAxisTitle(getString(R.string.bmi));
    }
}

File: app/src/main/java/com/example/tipper/Graph.kt
package com.example.tipper

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.jjoe64.graphview.GraphView
import com.jjoe64.graphview.LegendRenderer
import com.jjoe64.graphview.helper.StaticLabelsFormatter
import com.jjoe64.graphview.series.DataPoint
import com.jjoe64.graphview.series.PointsGraphSeries

class Graph : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.graph_layout)
        val graphView = findViewById<View>(R.id.graphView) as GraphView
        val staticLabelsFormatter = StaticLabelsFormatter(graphView)
        val series = PointsGraphSeries(
            arrayOf(
                DataPoint(0.0, 25.0),
                DataPoint(1.0, 22.0),
                DataPoint(2.0, 24.0),
                DataPoint(3.0, 26.0),
                DataPoint(4.0, 21.0),
                DataPoint(5.0, 20.0),
                DataPoint(6.0, 23.0)
            )
        )
        graphView.addSeries(series)
        staticLabelsFormatter.setHorizontalLabels(arrayOf("Jan", "Feb", "Mar", "Apr", "May", "Jun"))
        val gridLabelRenderer = graphView.gridLabelRenderer
        gridLabelRenderer.labelFormatter = staticLabelsFormatter
        series.color = R.color.colorPrimaryDark
        graphView.title = getString(R.string.graph_title)
        graphView.legendRenderer.isVisible = true
        graphView.legendRenderer.align = LegendRenderer.LegendAlign.BOTTOM
        gridLabelRenderer.horizontalAxisTitle = getString(R.string.month)
        gridLabelRenderer.verticalAxisTitle = getString(R.string.bmi)
    }
}